db/queries.py

```diff
@@ -1,46 +1,87 @@
+from datetime import datetime
+from typing import Any
+
+from django.db.models.functions import Concat
+
 from db.models import Message, User, Chat
-from django.db.models import Q, Count, F
+from django.db.models import Q, Count, F, Value
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(
+        text__icontains=word
+    ))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(
```

Comment:
cast to list because it returns queryset now, and return annotation is list[Chat]